runner/app/src/main/kotlin/org/promethist/core/repository/dynamodb/DynamoEventRepository.kt

```diff
@@ -2,28 +2,27 @@
 
 package org.promethist.core.repository.dynamodb
 
-import com.amazonaws.services.dynamodbv2.document.Item
 import com.amazonaws.services.dynamodbv2.document.KeyAttribute
 import com.amazonaws.services.dynamodbv2.document.spec.QuerySpec
 import org.jetbrains.kotlin.utils.addToStdlib.ifNotEmpty
 import org.litote.kmongo.Id
 import org.promethist.common.ObjectUtil
 import org.promethist.common.query.DynamoDbFiltersFactory
 import org.promethist.common.query.Query
+import org.promethist.common.repository.DynamoAbstractEntityRepository
 import org.promethist.core.model.DialogueEvent
-import org.promethist.core.model.Session
 import org.promethist.core.repository.EventRepository
 import java.util.*
 
 class DynamoEventRepository : DynamoAbstractEntityRepository<DialogueEvent>(), EventRepository {
 
-    private val dialogueEventTable by lazy { database.getTable(tableName("dialogueEvent")) }
+    override val table by lazy { database.getTable(tableName("dialogueEvent")) }
```

Comment:
It can be simplified and moved to abstract class, the only diff in implementations is table name.

In abstract class you can have
```
abstract val tableName:String
open val table by lazy { database.getTable(tableName(tableName)) }
```
and in implementation:
```
override val tableName = "dialogueEvent"
```